test/specs/modules/Checkbox/Checkbox-test.js

```diff
@@ -154,17 +142,24 @@ describe('Checkbox', () => {
       wrapperShallow(<Checkbox disabled />)
 
       wrapper.simulate('click')
-      wrapper
-        .find('input')
-        .should.not.be.checked()
+      wrapper.find('input').should.not.be.checked()
     })
+
     it('cannot be unchecked', () => {
       wrapperShallow(<Checkbox defaultChecked disabled />)
 
       wrapper.simulate('click')
-      wrapper
+      wrapper.find('input').should.be.checked()
+    })
+
+    it('is applied to the underlying html input element', () => {
```

Comment:
These are the tests, better to review in Split mode as GitHub did a diff mess because of the Lint